05.wc/wc.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+require 'optparse'
+
+def main
+  options = parse_options
+  if ARGV.empty?
+    handle_stdin(options)
+  else
+    handle_files(ARGV, options)
+  end
+end
+
+def parse_options
+  options = { lines: false, words: false, bytes: false }
+  OptionParser.new do |opts|
+    opts.banner = 'Usage: wc.rb [options] [file ...]'
+    opts.on('-l', 'Count lines') { options[:lines] = true }
+    opts.on('-w', 'Count words') { options[:words] = true }
+    opts.on('-c', 'Count bytes') { options[:bytes] = true }
+  end.parse!
+  options.values.any? ? options : options.transform_values { true }
+end
+
+def handle_stdin(options)
+  file_content = $stdin.read
```

Comment:
```suggestion
  content = $stdin.read
```

でいいかも（44行目も同様）
https://bootcamp.fjord.jp/pages/avoid-too-specific-var-name

58行目で`def count_all(content)`とメソッドを定義しているので、引数名と名前を合わせる目的もあります

Reply:
@JunichiIto 
ドキュメント共有ありがとうございます🙇‍♂️
`他の変数と区別しなくていいならシンプルな変数名を付ける`を気をつけて今後実装していきます